app/api/util.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+
+
+# returns the language codes from the django settings. ['de', 'en']
+def get_setting_lang_code():
+    return [lang[0] for lang in settings.LANGUAGES]
+
+
+# calls the function with the given attr and languages.
+# text_de, text_en
+def model_lang_fields(obj, attr, lang):
```

Comment:
I would call `get_setting_lang_codes()` instead of accepting the lang code list here. Also, please use plurals for lists and other collections or collection-returning-functions :)

Then, please convert the comment to docstring format instead (see example above). and.. which function?